JPS_Ontology/ontology/ontobatchexperiment/OntoBatchExperiment.csv

```diff
@@ -0,0 +1,15 @@
+Source,Type,Target,Relation,Domain,Range,Quantifier,Comment,Defined By,Label
+OntoBatchExperiment,TBox,https://www.theworldavatar.com/kg/ontobatchexperiment/,https://www.w3.org/2007/05/powder-s#hasIRI,,,,,,
+OntoBatchExperiment,TBox,1,http://www.w3.org/2002/07/owl#versionInfo,,,,,,
+OntoBatchExperiment,TBox,OntoBatchExperiment is an ontology to represent batch experiments and relevant information.,http://www.w3.org/2000/01/rdf-schema#comment,,,,,,
+BatchExperiment,Class,https://www.theworldavatar.com/kg/ontoreaction/ReactionExperiment,IS-A,,,,A type of experiment or process in which a fixed quantity of materials is processed together in a single batch or lot.,https://www.theworldavatar.com/kg/ontobatchexperiment/,Batch experiment
+BatchExperimentCategory,Class,,,,,,A classification or type of batch experiment based on specific criteria or characteristics.,https://www.theworldavatar.com/kg/ontobatchexperiment/,Batch experiment category
+Identifier,Class,,,,,,"A unique identifier assigned to a batch experiment that is derived from other types of information (e.g. work code, batch number, initials, BatchExperimentCategory).",https://www.theworldavatar.com/kg/ontobatchexperiment/,Identifier
+WorkCode,Class,,,,,,A confidential internal identifier assigned to a batch experiment.,https://www.theworldavatar.com/kg/ontobatchexperiment/,Work code
+BatchNumber,Class,,,,,,A number assigned to a batch experiment.,https://www.theworldavatar.com/kg/ontobatchexperiment/,Batch number
+hasBatchExperimentCategory,Object Property,,,BatchExperiment,BatchExperimentCategory,,A relationship to describe the category of a batch experiment.,https://www.theworldavatar.com/kg/ontobatchexperiment/,has batch experiment category
+https://www.theworldavatar.com/kg/ontoderivation/isDerivedFrom,Object Property,,,https://www.theworldavatar.com/kg/ontoderivation/Derivation,WorkCode UNION BatchNumber UNION https://www.theworldavatar.com/kg/ontoprofile/Initials UNION BatchExperimentCategory,,A relationship to describe an identifier being derived from other types of information.,https://www.theworldavatar.com/kg/ontobatchexperiment/,is derived from
```

Comment:
The range of `ontoderivation:isDerivedFrom` is already `owl:Thing`, so this line is redundant. Actually, this line contradicts the original definition. Either way, please remove.